### ISSUE TITLE
Sekaikomik: Blocked by Cloudflare

### ISSUE THREAD

**Issue by Chan220** [tachiyomiorg/tachiyomi-extensions#8892]
### Source information

Sekaikomik 1.2.19

### Steps to reproduce

1. open
2. eror 403
3.  open in browser
4. blocked by cloudflare

![47344e2c-66fd-4a21-af8c-800c81ee2cd2](https://user-images.githubusercontent.com/81818094/137435329-64d9de0d-d853-4534-ae0f-1d198a81d282.jpg)

### Expected behavior

open normally

### Actual behavior

blocked by cloudflare

### Tachiyomi version

0.12.1

### Android version

Android 11

### Other details

blocked by cloudflare

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Chan220** [comment]
thanks bro

**Comment by Chan220** [comment]
hmmm
no updates

**Comment by alessandrojean** [comment]
There's no ETA when this is going to be fixed. Wait until someone have free time and wants to work on it.

**Comment by Chan220** [comment]
ok

> There's no ETA when this is going to be fixed. Wait until someone have free time and wants to work on it.
> 
> Also, it's not possible to fix Cloudflare blocks in the extension side, you need to wait for the website to lower the protection.

ok

**Comment by Chan220** [comment]
[](url
![47344e2c-66fd-4a21-af8c-800c81ee2cd2](https://user-images.githubusercontent.com/81818094/137434226-77167f22-7c27-4638-b360-785fb97629c6.jpg)
)

**Comment by Chan220** [comment]
> There's no ETA when this is going to be fixed. Wait until someone have free time and wants to work on it.
> 
> Also, it's not possible to fix Cloudflare blocks in the extension side, you need to wait for the website to lower the protection.

ok

**Comment by arkon** [comment]
Refer to the troubleshooting guide (also detailed in #3475).
